mathics/doc/latex/sed-hack.sh

```diff
@@ -36,6 +36,7 @@ sed -i -e 's/⧦/\\\\Equiv/g' documentation.tex
 sed -i -e 's/⊻/xor/g' documentation.tex
 sed -i -e 's/∧/&&/g' documentation.tex
 sed -i -e 's/‖/||/g' documentation.tex
+sed -i -e 's/ⅅ/$\\mathbb{D}$/' documentation.tex
```

Comment:
Good! Can the other sed's get removed?

Reply:
I have to check which one of these can be removed.